src/clusterfuzz/_internal/crash_analysis/stack_parsing/stack_symbolizer.py

```diff
@@ -525,6 +528,61 @@ def _lkl_line_parser(self, line):
 
     return None, None, None, None, None
 
+  def _trusty_line_parser(self, line):
+    """Parses line for memory_space, offset, and addr"""
+    match = STACK_TRACE_LINE_TRUSTY.match(line)
+    if match:
+      memory_space, offset, addr = match.groups()
+      return memory_space, offset, addr
+
+    return None, None, None
+
+  def _extract_trusty_app_name(self, stacktrace):
```

Comment:
Nit: please add docstrings for all of these methods.

Reply:
Done - the docstrings should be added now.